packages/ghc.rb

```diff
@@ -16,7 +16,7 @@ class Ghc < Package
   depends_on 'libtinfo'
 
   def self.build
-    system "stack setup ghc-8.4.2"
+    system "stack setup ghc-8.6.5"
```

Comment:
`system "stack setup ghc-#{version}"`?  This would make the next updates easier since we would only change the version in one place above.